tests/test_pypi.py

```diff
@@ -809,13 +820,22 @@ def test_download_pkg_sdist(pkg_pytest):
 
 def test_ciso_recipe():
     recipe = GrayskullFactory.create_recipe(
-        "pypi", Configuration(name="ciso", version="0.1.0")
+        "pypi", Configuration(name="ciso", version="0.2.2")
     )
     assert sorted(recipe["requirements"]["host"]) == sorted(
-        ["cython", "numpy", "pip", "python"]
+        [
+            "cython >=3",
+            "numpy >=2.0.0rc1",
+            "oldest-supported-numpy",
```

Comment:
We shouldn't have both `numpy >=2.0.0rc1` and `oldest-supported-numpy` but it looks like grayskull doesn't support selectors?

Reply:
which selectors are you talking about? It does support a few of them but in this case they might not be being generated for some reason

Reply:
Yeah this is `requirements/host`. Don't think we want `oldest-supported-numpy` here

Reply:
see my comment here ;)
https://github.com/conda/grayskull/pull/552/files#r1721631804

Reply:
that equal over there is just a syntax sugar, but if you properly compare the ``numpy`` and ``oldest-supported-numpy`` you will see the selectors :)

Reply:
Ah! Indeed, looking at the generated meta.yaml, I see:

```
    - oldest-supported-numpy  # [py<39]
    - numpy >=2.0.0rc1  # [py>=39]
 ```